### DISCUSSION TITLE
[Taiko mod]: Colorblind

### DISCUSSION THREAD

**Discussion by Ghost1928563945** [ppy/osu#22229] [Ideas]
What if we made a taiko mode such that you just have to hit keys, no dons or kas just hit whatever when there is a note.

**Comment by reo-the-enby** [comment]
The relax* mode does that

**Reply by bdach** [reply]
Also https://github.com/ppy/osu/discussions/17058.

I'm not actually sure if the EZ mod will currently count any keypress as a hit though. It doesn't seem to, based on source.

**Reply by bdach** [reply]
It is actually the relax mod that does this. It's yet to be reimplemented (https://github.com/ppy/osu/issues/7914).

**Reply by reo-the-enby** [reply]
oops, wrote the wrong mod

**Reply by Ghost1928563945** [reply]
@bdach why can't it be a seperate mod, i don't want whatever realax mod does enless it just does this.

**Reply by Ghost1928563945** [reply]
its too bad mods can't be added in rulesets

**Reply by Rekunan** [reply]
@NullifiedJosh relax is the mod that does this, it's just not implemented yet.

**Comment by TheFearlessDeath500** [comment]
Don’t make this like a separate mod. What you just mentioned was just the Doron/Stealth mod from official Taiko games. It hides the notes like what you mentioned and I feel like it would fit more for a mod similar to “Hidden”.

**Reply by Rekunan** [reply]
Are you proposing to add it as an option under the hidden mod? I think it's actually better to have it as a separate mod, like how stable has it as the relax mod.

**Comment by TomyDoesThings** [comment]
I believe a better idea to a Taiko Colorblind mod would be to make all the notes lose their colors (they maybe become grey), and so, you have to memorize which are dons and katsus. It would be a neat memorization mod.

**Reply by nagi-desuuu** [reply]
This is the better idea than the original poster ever had.

**Reply by cdwcgt** [reply]
![result](https://github.com/ppy/osu/assets/34775378/2f51ea29-7680-4e01-a19f-646135743c37)
This should be what is wanted, but my changes don't work on the argon skin....
its AccentColour has multiple colors, I don’t know how to handle it properly
